examples/dreambooth/train_dreambooth.py

```diff
@@ -509,14 +510,17 @@ def main(args):
         revision=args.revision,
     )
 
-    if is_xformers_available():
-        try:
-            unet.enable_xformers_memory_efficient_attention()
-        except Exception as e:
-            logger.warning(
-                "Could not enable memory efficient attention. Make sure xformers is installed"
-                f" correctly and a GPU is available: {e}"
-            )
+    if args.use_xformers:
+        if is_xformers_available():
+            try:
+                unet.enable_xformers_memory_efficient_attention()
+            except Exception as e:
+                logger.warning(
+                    "Could not enable memory efficient attention. Make sure xformers is installed"
+                    f" correctly and a GPU is available: {e}"
+                )
+        else:
+            logger.warning("Could not enable memory efficient attention. Make sure xformers is installed correctly.")
```

Comment:
```suggestion
    if args.use_xformers:
            unet.enable_xformers_memory_efficient_attention()
```

I agree with @patrickvonplaten here, if the user explicilty requests they want `xformers`, let's fail if it's not available (it's important that they know it's not working in this case).